pinning-options.txt

```diff
@@ -1,14 +1,14 @@
- .---------------.
- | domain CA (1) |
- '---------------'
+ .------------.
+ | pub-CA (1) |
```

Comment:
Why is domain CA renamed to pub-CA? Most typical domain CAs are not public i.e. not part of PKIX. Rather, they use a private root CA certificate typically.